lib/puppet/provider/logical_volume/lvm.rb

```diff
@@ -221,6 +221,22 @@ def size=(new_size)
     end
   end
 
+  def stripes
+    # Run the lvs command with the -o option to get only the stripes count
+    raw = (lvs '-o', 'stripes', '--noheadings', path)
+
+    output_array = raw.strip
+    output_array
+  end
+
+  def stripes=(new_stripes_count)
```

Comment:
what's the purpose of this setter? Is that used anywhere?

Reply:
Hey @bastelfreak , I was trying to not allow the user to change the stripes of an existing lvm so added setter to raise an error. Referred the implementation the way mirror & mirror= getter & setter handled. Please let me know if there is better way to do this happy to comply...

Reply:
ah okay. that was my impression as well. But than why did you change it to a gettter and setter at all?

Reply:
I could only think of this way to handle it, can you suggest how to do this without getter and setter?

Reply:
Discussed in DM, hence resolving it...